### ISSUE TITLE
Wunder bar - shift + right click to reset character data

### ISSUE THREAD

**Issue by JazzyJones** [Toxicom/toxiui#111]
### Description
Wunder bar - shift + right click to reset character data

### Lua error
```lua
Message: ...vUI_ToxiUI/Modules/WunderBar/SubModules/Currency.lua:120: attempt to call upvalue 'EasyMenu' (a nil value)
Time: Wed Jul 24 19:31:24 2024
Count: 1
Stack: ...vUI_ToxiUI/Modules/WunderBar/SubModules/Currency.lua:120: attempt to call upvalue 'EasyMenu' (a nil value)
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/WunderBar/SubModules/Currency.lua"]:120: in function `OnClick'
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/WunderBar/Modules.lua"]:126: in function `ModuleOnClick'
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/WunderBar/Modules.lua"]:289: in function <...ce/AddOns/ElvUI_ToxiUI/Modules/WunderBar/Modules.lua:288>

Locals: self = <table> {
 modules = <table> {
 }
 totalGold = 858905799
 defaultModuleLibraries = <table> {
 }
 Initialized = true
 currencyText = FontString {
 }
 name = "ElvUI_ElvUI_ToxiUI_WunderBar_Currency"
 orderedModules = <table> {
 }
 freeBagSpace = 75
 defaultModuleState = true
 frame = Frame {
 }
 baseName = "ElvUI_ToxiUI"
 SubModuleHolder = Frame {
 }
 Module = TXUIWunderBarRightPanelModule1 {
 }
 Data = <table> {
 }
 db = <table> {
 }
 goldSpent = 0
 enabledState = true
 currencyIcon = FontString {
 }
 isEnabled = false
 moduleName = "Currency"
 goldProfit = 0
}
_ = TXUIWunderBarRightPanelModule1 {
 parent = TXUIWunderBarRightPanel {
 }
 extendedModule = false
 calculatedSize = 0
 parentName = "RightPanel"
 moduleIndex = 7
 parentIndex = 1
 extendModuleRequest = false
 subModule = <table> {
 }
}
button = "RightButton"
menuList = <table> {
 1 = <table> {
 }
 2 = <table> {
 }
 3 = <table> {
 }
 4 = <table> {
 }
 5 = <table> {
 }
 6 = <table> {
 }
 7 = <table> {
 }
 8 = <table> {
 }
}
(*temporary) = nil
(*temporary) = <table> {
 1 = <table> {
 }
 2 = <table> {
 }
 3 = <table> {
 }
 4 = <table> {
 }
 5 = <table> {
 }
 6 = <table> {
 }
 7 = <table> {
 }
 8 = <table> {
 }
}
(*temporary) = ElvUI_EasyMenu {
 relativeTo = Frame {
 }
 point = "BOTTOMRIGHT"
 xOffset = 0
 Right = ElvUI_EasyMenuRight {
 }
 Left = ElvUI_EasyMenuLeft {
 }
 yOffset = 0
 relativePoint = "TOPRIGHT"
 Button = ElvUI_EasyMenuButton {
 }
 Text = ElvUI_EasyMenuText {
 }
 Middle = ElvUI_EasyMenuMiddle {
 }
 Icon = ElvUI_EasyMenuIcon {
 }
}
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "MENU"
(*temporary) = "attempt to call upvalue 'EasyMenu' (a nil value)"
_G = <table> {
 ERR_OUT_OF_CHI = "Not enough chi"
 DH_HAVOC_CORE_ABILITY_2 = "Strong melee attack that consumes Fury. If it critical strikes, some Fury is refunded."
 UNIT_NAMES_COMBATLOG_TOOLTIP = "Color unit names."
 LE_GAME_ERR_CHAT_RAID_RESTRICTED_TRIAL = 783
 SPELL_FAILED_CUSTOM_ERROR_71 = "This partygoer wants to dance with you."
 LE_GAME_ERR_PET_SPELL_TARGETS_DEAD = 421
 ClassTrialTimerDisplayMixin = <table> {
 }
 MultiBarLeftButton7 = MultiBarLeftButton7 {
 }
 LE_GAME_ERR_INVALID_FOLLOW_PVP_COMBAT = 375
 MerchantItem7AltCurrencyFrameItem1Text = MerchantItem7AltCurrencyFrameItem1Text {
 }
 OPTION_SHOW_ACTION_BAR5_TOOLTIP = "Attached to the left side of Action Bar 4 by default"
 BINDING_NAME_NAMEPLATES = "Show Enemy Nameplates"
 INSTANCE_UNAVAILABLE_OTHER_TEMPORARILY_DISABLED = "%s cannot enter. This instance is temporarily disabled."
 COVENANT_MISSIONS_HEALTH = "Health"
 PVPReadyDialogBottomArt = PVPReadyDialogBottomArt {
 }
 ERR_NOAMMO_S = "%s"
 BOOST2_WARRIOR_COLOSSUSSMASH = "Use Colossus Smash.

Colossus Smash increases your damage."
 AceConfigDialogTooltipTexture20 = AceConfigDialogTooltipTexture20 {
 }
 CinematicFrameRaidBossEmoteFrame = CinematicFrameRaidBossEmoteFrame {
 }
 LE_GAME_ERR_ONLY_ONE_QUIVER = 36
 LOSS_OF_CONTROL_DISPLAY_FEAR = "Feared"
 Boss1TargetFrameValue = Boss1TargetFrameValue {
 }
 ElvUI_Bar6Button6 = ElvUI_Bar6Button6 {
 }
 AUCTIONATOR_L_DEFAULT_LIST_HEADER = "Default Shopping List"
 ACTION_SPELL_MISSED_POSSESSIVE = "1"
 RaidUtility_CloseButton = RaidUtility_CloseButton {
 }
 LE_GAME_ERR_AZERITE_ESSENCE_SELECTION_FAILED_ESSENCE_NOT_UNLOCKED = 1074
 DUNGEON_FLOOR_DRAGONBLIGHTCHROMIESCENARIO2 = "Andorhal"
 ElvUIReage
```

### ToxiUI Status Report
![image](https://github.com/user-attachments/assets/6ee3bb73-6d46-4c0e-9a34-8eef855225e8)

### Screenshots
![image](https://github.com/user-attachments/assets/10262165-ea1d-4984-ab22-969eca91461c)

### To reproduce
If this issue is describing a bug, include some steps to reproduce the behavior

1. Press shift + right click to reset character data on wunderbar and lua error should pop up.

**Comment by Toxicom** [comment]
7edc0edd00479b87fc1b5778d0903bd1bb3af45e

**Comment by Toxicom** [comment]
Should be fixed in [v6.7.4](https://github.com/Toxicom/toxiui/releases/tag/v6.7.4)
